### DISCUSSION TITLE
problems with installation

### DISCUSSION THREAD

**Discussion by voztotal** [Eudorajab1/jasmin_smsc_gui#7] [Q&A]
I am trying to install the gui interface, but I have this error.

`Status: Downloaded newer image for jookies/jasmin:0.10
Creating smppsim            ... done
Creating jasmin_rabbit-mq_1 ... done
Creating jasmin_redis_1     ... done
Creating jasmin             ... 
Creating jasmin             ... error

ERROR: for jasmin  Cannot start service jasmin: driver failed programming external connectivity on endpoint jasmin (fd47b6479b15d9d51cd81a96c9856a5d5fd3ee290d5662bf1ad63ecadc04bcf8): Bind for 0.0.0.0:8990 failed: port is already allocated

ERROR: for jasmin  Cannot start service jasmin: driver failed programming external connectivity on endpoint jasmin (fd47b6479b15d9d51cd81a96c9856a5d5fd3ee290d5662bf1ad63ecadc04bcf8): Bind for 0.0.0.0:8990 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.`

I already have jasmin running with docker, can you give me any idea how to solve it?

**Comment by Eudorajab1** [comment]
From the trace supplied it looks like port 8990 is already in use by your existing Jasmin instance. So there are a couple of options if you installing user docker compose. If you already have Jasmin installed you just need to point the GUI to your existing Jasmin docker instance instead of trying to install it again (using default ports) or you can take the slightly longer route and bind to different ports and change the config files.
Hope this helps and as a first step I would suggest stopping your existing docker installation (which will free up the default ports like 8990) and re-run the install

**Reply by voztotal** [reply]
It already works, I saw in your post that you have a paid platform, can you give me more information?

**Comment by Eudorajab1** [comment]
So essentially the paid platform allows you to manage multiple Jasmin instances and has a bit more functionality. I also have a SMS gateway solution that I need to update which allows you to mange customers, clients, billing, connections to operators or third party providers etc
basically a full solution to manage sms business (GUI is just one module for managing Jasmin) 
If you let me know a bit more about what you are looking for we can see if there is a fit if you are interested.

**Comment by voztotal** [comment]
I just need a platform to manage rates, carriers, clients, the closest thing I found was jasmin, but I'm interested in knowing your platform

**Comment by Eudorajab1** [comment]
Are you going to be dealing with re-seller type clients as well or?

**Comment by voztotal** [comment]
Not at the moment, but it is a functionality that I am looking for in the future.

**Comment by Eudorajab1** [comment]
okay let me prepare something for you over the next few days and we can discuss

**Comment by voztotal** [comment]
thx, :D
